src/main/java/gmail/mishchuk/lesson40_FI/LogicToFI.java

```diff
@@ -0,0 +1,56 @@
+package gmail.mishchuk.lesson40_FI;
+
+import java.util.Random;
+
+public class LogicToFI {
+
+//    public static boolean getPrimeNumberElvisStyle (int num) {
+//        if (num <= 1 ? false);
+//        for (int i = 2; i * i <= num; i++) {
+//            if (num % i == 0 ? false : true);
+//            }
+//        return true;
+//    }
+
+    public static boolean getPrimeNumber(int numberToCheck) {
+        if (numberToCheck <= 1) {
+            return false;
+        }
+        for (int i = 2; i * i <= numberToCheck; i++) {
+            if (numberToCheck % i == 0) {
+                return false;
+            }
+        }
+        return true;
+    }
```

Comment:
Ви все зробили як методи, а я хотів би щоб Ви зробили все через функціональні інтерфейси 😄 

Будь-ласка переробіть інші також та змініть тести відповідно до змін

Приклад:

```suggestion
    public static Predicate<Integer> isPrimeNumber = numberToCheck -> {
        if (numberToCheck <= 1) {
            return false;
        }
        for (int i = 2; i * i <= numberToCheck; i++) {
            if (numberToCheck % i == 0) {
                return false;
            }
        }
        return true;
    };
```